mpf/devices/driver.py

```diff
@@ -318,3 +347,33 @@ def pulse(self, pulse_ms: int = None, pulse_power: float = None, max_wait_ms: in
             self._pulse_now(pulse_ms, pulse_power)
 
         return wait_ms
+
+    @event_handler(4)
+    def event_timed_enable(self, hold_ms: int = None, hold_power: float = None, pulse_ms: int = None, pulse_power: float = None,  max_wait_ms: int = None, **kwargs):
+        """Event handler for timed enable events."""
+        del kwargs
+        self.timed_enable(hold_ms, hold_power, pulse_ms, pulse_power, max_wait_ms)
+
+    def timed_enable(self, hold_ms: int = None, hold_power: float = None, pulse_ms: int = None, pulse_power: float = None, max_wait_ms: int = None):
+        """Pulse and enable this driver.
+
+        Args:
+        ----
+            enable_duration_ms: The number of milliseconds the driver should be
```

Comment:
Parameters are inconsistent here. enable_duration_ms is not a parameter.

Reply:
Yes, the interchangeability of `hold` and `enable` throws me off sometimes :)

Should we consider migrating the "hold"-based config values to "enabled"? Like `default_enable_power` and `kwargs["enable_ms"]`? With fallback logic to not break existing games, but a log notice that the "hold" params are going to be deprecated?